kokoro/gcp_windows/kokoro_build.bat

```diff
@@ -25,9 +25,11 @@ call conan package -bf %REPO_ROOT%\build\ %REPO_ROOT%
 if %errorlevel% neq 0 exit /b %errorlevel%
 
 for %%g in ("%KOKORO_JOB_NAME:/=" "%") do set build_type=%%g
+for /f "delims=/" %%g in ("%KOKORO_GITHUB_PULL_REQUEST_TARGET_BRANCH%") do set target_branch=%%g
 set symbol_uploading_required=false
 if %build_type% == "release" set symbol_uploading_required=true
 if %build_type% == "nightly" set symbol_uploading_required=true
+if %build_type% == "continuous" if %target_branch% == "release" set symbol_uploading_required=true
```

Comment:
I guess this won't work like that. What about `if %build_type% == "continuous_on_release_branch" set symb...`?

Reply:
Fixed it for Windows and added for ubuntu_ggp as well.